README.md

```diff
@@ -86,10 +86,15 @@ Reading from the JSON formatted log files with `in_tail` and wildcard filenames:
   @type tail
   path /var/log/containers/*.log
   pos_file fluentd-docker.pos
-  time_format %Y-%m-%dT%H:%M:%S
-  tag kubernetes.*
-  format json
   read_from_head true
+  tag kubernetes.*
+  <parse>
+    @type json
+    time_key time
+    time_type string
+    time_format "%Y-%m-%dT%H:%M:%S.%NZ"
+    keep_time_key false
+  </parse>
 </source>
```

Comment:
We should support CRI-o log file format too with our example.  Please change the `<parse>` block to something like this (or I can change the commit directly if you don't mind):
```
  <parse>
    @type multi_format
    <pattern>
      format json
      time_key time
      time_type string
      time_format "%Y-%m-%dT%H:%M:%S.%NZ"
      keep_time_key false
    </pattern>
    <pattern>
      format regexp
      expression /^(?<time>.+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/
      time_format '%Y-%m-%dT%H:%M:%S.%N%:z'
      keep_time_key false
    </pattern>
  </parse>
```

Reply:
I have been search all over the place for this. Not nearly reachable enough.